activelearning/dataset/ocp.py

```diff
@@ -25,6 +26,15 @@ def __init__(
         self.appended_data = []
         self.return_target = return_target
 
+        # https://github.com/RolnickLab/ocp/blob/c93899a23947cb7c1e1409cf6d7d7d8b31430bdd/ocpmodels/trainers/base_trainer.py#L361
+        # https://github.com/RolnickLab/ocp/blob/c93899a23947cb7c1e1409cf6d7d7d8b31430bdd/configs/models/tasks/is2re.yaml#L25
+        # https://github.com/RolnickLab/ocp/blob/c93899a23947cb7c1e1409cf6d7d7d8b31430bdd/ocpmodels/modules/normalizer.py#L11
+        self.target_normalizer = Normalizer(
+            mean=-1.525913953781128,
+            std=2.279365062713623,
```

Comment:
I would add these numbers as "global variables" in the script, together with the references.

Reply:
updated